pip_audit/_cli.py

```diff
@@ -286,7 +285,6 @@ def _parser() -> argparse.ArgumentParser:  # pragma: no cover
         type=str,
         help="base URL of the Python Package Index; this should point to a repository compliant "
         "with PEP 503 (the simple repository API)",
-        default=PYPI_URL,
```

Comment:
I think we want to keep this `default` in the CLI -- behaviorally it should remain the default, and having it appear as such in the `--help` render makes for a nicer UX 🙂

Reply:
So leaving this as the default means that it will always be explicitly passed to the `pip install` CLI, which results in the `index-url` specified in the `pip.conf` being ignored. I understand that it makes for a more friendly UX to specify as such, but would it be reasonable to convey that the url will be resolved by `pip` if not passed by the user explicitly?

If this is a deal breaker though, then I'll have to dig deeper into `pip` to determine how it is performing the `index-url` resolution.